src/providers/RelayProvider.ts

```diff
@@ -1,9 +1,13 @@
+import { BigNumber } from 'bignumber.js'
+import { ethers } from 'ethers'
 import { Observable } from 'rxjs/Observable'
 
 import { TLProvider } from './TLProvider'
 
 import utils from '../utils'
 
+import { TxInfos, TxInfosRaw } from '../typings'
+
 export class RelayProvider implements TLProvider {
```

Comment:
Why does this have to implement both and not just the ethers provider. I think we might mix concepts here, lets have a call tomorrow.

Reply:
I wasn't 100% sure about this. I thought that if we want to support other providers like `web3`. `etherscan`, ... it would be necessary to have another abstraction `TLProvider`. But let us talk about this in the call.